lib/dcd-provider.coffee

```diff
@@ -17,12 +17,16 @@ module.exports =
   updateClientCommand: ->
     @clientCommand = atom.config.get(packageName + ".dcdClient")
 
+  updateDubCommand: ->
+    @dubCommand = atom.config.get(packagename + ".dub")
```

Comment:
Small typo here : it should be `packageName` instead of `packagename`. After correcting this, it seems to be working fine